### ISSUE TITLE
Restify middleware: HTTP methods mapping is wrong

### ISSUE THREAD

**Issue by banzsh** [theganyo/swagger-node-runner#20]
When `DELETE` operation is specified in `swagger.yaml` the Restify middleware tries to invoke `app.delete` which is not an actual function in Restify (it uses `app.del` instead). This might be the same for `OPTIONS` which in Restify is `app.opts`. 

In previous version it was working for iteration was done over Restify's methods and now you are iterating over the methods described in `swagger.yaml`

As a hack I use `app.delete = app.del` in my app.js.

Exception stack:

```
Error in callback! Tossing to global error handler. TypeError: app[method] is not a function
    at registerHandler (/home/api/node_modules/swagger-restify-mw/node_modules/swagger-node-runner/lib/restify_middleware.js:24:18)
    at /home/banzsh/projects/api/node_modules/swagger-restify-mw/node_modules/swagger-node-runner/lib/restify_middleware.js:33:11
    at Array.forEach (native)
    at /home/banzsh/projects/api/node_modules/swagger-restify-mw/node_modules/swagger-node-runner/lib/restify_middleware.js:32:31
    at Array.forEach (native)
    at Middleware.register (/home/banzsh/projects/api/node_modules/swagger-restify-mw/node_modules/swagger-node-runner/lib/restify_middleware.js:30:21)
    at /home/banzsh/projects/api/app.js:76:20
    at /home/banzsh/projects/api/node_modules/swagger-restify-mw/lib/index.js:32:5
    at /home/banzsh/projects/api/node_modules/swagger-restify-mw/node_modules/swagger-node-runner/index.js:234:7

```

**Comment by theganyo** [comment]
Thanks for reporting this, #21, and #22. It looks like perhaps I should revert to the global registration method I was using before for Restify instead of individually registering the paths. Had you seen any issues with that method?

**Comment by banzsh** [comment]
Actually I've tried this on the current version having `app[method]('.*' ...)` and that worked fine. Basically I think the former implementation was simpler and working. IMO this would fix #21 and #22 as well.

Thank you!
